--- libprelude-3.1.0/configure.ac.orig	2016-09-15 08:49:10.732000884 +0200
+++ libprelude-3.1.0/configure.ac	2016-10-09 15:06:53.560791749 +0200
@@ -565,6 +565,7 @@
        [PKG_CHECK_MODULES(LUA, [lua_ver >= 5.1], [
           AC_DEFINE([HAVE_LUA], [1], [liblua])
           AC_DEFINE([HAVE_LUA_H], [1], [lua.h])
+	  LUA_INTERP=$(pkg-config --variable=interpreter lua_ver)
           lua_pkg_found=1
        ],[:])]
    )
@@ -582,7 +583,7 @@
       AC_CHECK_HEADERS([lua.h lauxlib.h], with_lua=yes, [with_lua=no; break])
       CPPFLAGS="$old_CPPFLAGS"
 
-      LUA_VERSION=$(lua -e "print(string.sub(_VERSION, 5))")
+      LUA_VERSION=$($LUA_INTERP -e "print(string.sub(_VERSION, 5))")
       AC_SUBST(LUA_VERSION)
 
    else
