diff -ur libprelude-1.0.1.orig/bindings/libpreludecpp.i libprelude-1.0.1.new/bindings/libpreludecpp.i
--- libprelude-1.0.1.orig/bindings/libpreludecpp.i	2012-06-01 15:12:00.000000000 +0000
+++ libprelude-1.0.1.new/bindings/libpreludecpp.i	2013-05-17 12:58:32.606635634 +0000
@@ -22,6 +22,9 @@
 *****/
 
 %module PreludeEasy
+%{
+#include <stddef.h>
+%}
 
 %include "std_string.i"
 %include "std_vector.i"
