--- libprelude-0.9.17.2/configure.in.orig	2008-05-06 17:51:17.000000000 +0200
+++ libprelude-0.9.17.2/configure.in	2008-07-16 22:08:23.058034707 +0200
@@ -226,8 +226,10 @@
 if test x$with_system_ltdl != xno; then
         CPPFLAGS_bkp=$CPPFLAGS
         LDFLAGS_bkp=$LIBS
+	if x$with_system_ltdl != x -a x$with_system_ltdl != x/usr ; then
         CPPFLAGS="-I$with_system_ltdl/include"
         LDFLAGS="-L$with_system_ltdl/lib"
+	fi
 
         AC_CHECK_HEADER(ltdl.h, AC_CHECK_LIB(ltdl, lt_dlopen, have_system_ltdl=yes))
 
@@ -244,8 +246,13 @@
         AC_CONFIG_SUBDIRS(libltdl)
 else
         with_ltdl_type="System"
+	if test -n "$with_system_ltdl" -a "$with_system_ltdl" != "yes"; then
         INCLTDL="-I$with_system_ltdl/include"
         LIBLTDL="-L$with_system_ltdl/lib -lltdl"
+	else
+        INCLTDL=
+        LIBLTDL="-lltdl"
+	fi
 fi
 
 AM_CONDITIONAL(HAVE_SYSTEM_LIBLTDL, test x$have_system_ltdl = xyes)
