--- libprelude-1.0.1/bindings/low-level/python/Makefile.am~	2012-04-02 11:41:15.000000000 +0200
+++ libprelude-1.0.1/bindings/low-level/python/Makefile.am	2015-12-27 20:54:02.339946498 +0100
@@ -8,7 +8,7 @@
 	CC="$(BINDINGS_CC)" $(PYTHON) setup.py build
 
 install-exec-hook:
-	$(PYTHON) setup.py install `test -n "$(DESTDIR)" && echo --root $(DESTDIR)`
+	$(PYTHON) setup.py install `test -n "$(DESTDIR)" && echo --root $(DESTDIR)` --prefix=$(prefix)
 
 uninstall-hook: 
 	$(PYTHON) setup.py uninstall `test -n "$(DESTDIR)" && echo --root $(DESTDIR)`
--- libprelude-1.0.1/bindings/python/Makefile.am~	2012-04-02 11:41:15.000000000 +0200
+++ libprelude-1.0.1/bindings/python/Makefile.am	2015-12-27 20:53:20.153594859 +0100
@@ -9,7 +9,7 @@
 	CC="$(CXX)" $(PYTHON) setup.py build
 
 install-exec-hook:
-	$(PYTHON) setup.py install `test -n "$(DESTDIR)" && echo --root $(DESTDIR)`
+	$(PYTHON) setup.py install `test -n "$(DESTDIR)" && echo --root $(DESTDIR)` --prefix=$(prefix)
 
 uninstall-hook:
 	$(PYTHON) setup.py uninstall `test -n "$(DESTDIR)" && echo --root $(DESTDIR)`
